plutus-core/plutus-ir/test/ParserSpec.hs

```diff
@@ -77,7 +77,7 @@ parseProg :: T.Text
     -> Either
         ParserErrorBundle
         (Program TyName Name PLC.DefaultUni PLC.DefaultFun SourcePos)
-parseProg = parse program "test"
+parseProg = parseProgram
```

Comment:
I need to specify the type for `parseProgram` so I took it out to avoid a long signature in the functions. It's a global binding because it's used in a few places. I'll add a comment about needing to specify the sig.